mypy/semanal_enum.py

```diff
@@ -107,9 +107,11 @@ def parse_enum_call_args(self, call: CallExpr,
         if len(args) < 2:
             return self.fail_enum_call_arg("Too few arguments for %s()" % class_name, call)
         if len(args) > 2:
-            return self.fail_enum_call_arg("Too many arguments for %s()" % class_name, call)
-        if call.arg_kinds != [ARG_POS, ARG_POS]:
-            return self.fail_enum_call_arg("Unexpected arguments to %s()" % class_name, call)
+            if call.arg_kinds != [ARG_POS, ARG_POS, ARG_NAMED_OPT]:
```

Comment:
Enum can accept up to [four named arguments](https://docs.python.org/3/library/enum.html#functional-api), not just one.

You should also try checking `call.arg_names` to make sure the arguments are what we expect.

Reply:
@Michael0x2a Sure! Will do it.